### PR TITLE
add demo deployment with rbac

### PR DESCRIPTION
**Pull request by zgfh**
yaml: add demo deployment with rbac

this can setup operator easily by `kubectl create -f deployment-with-rbac.yaml`

[skip ci]

### CODE REVIEW

File: example/deployment-with-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# This is your Editor pane.
```
Comment (hexfusion) [inline example/deployment-with-rbac.yaml]: Please remove this first line.

File: example/deployment-with-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+kind: Namespace
+metadata:
+  name: etcd-operator
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcd-operator
+  namespace: etcd-operator
+
```
Comment (hexfusion) [inline example/deployment-with-rbac.yaml]: For consistency can we add a line break after each section or remove this line.

**Comment by zgfh** [comment]
@hexfusion  sorry ,fixed

**Comment by zgfh** [comment]
@hexfusion ping ,please have a look again,thanks

**Pull request closed without merging**